helm-chart-scylla/README.md

```diff
@@ -0,0 +1,115 @@
+# ScyllaDB Helm Chart on GKE
+
+ ## What is Helm Charts? Why use a Helm Chart? 
+  
+Helm uses a packaging format called charts. A chart is a collection of files that describe a related set of Kubernetes resources. A single chart might be used to deploy something simple, like a memcached pod, or something complex, like a full web app stack with HTTP servers, databases, caches, and so on. 
+
+Helm Charts deploys all the kubernetes entities in a ordered fashion wrapping them together in a RELEASE. In addition to that you also get versioning control allowing you to upgrade your release and rolling back changes.
```

Comment:
Suggestion: "In addition, you also get version control of releases. This allows you to upgrade a release and roll-back changes."

Reply:
Good stuff.